test/integration/main-esapi.c

```diff
@@ -130,7 +157,7 @@ tcti_proxy_initialize(
     /* Init TCTI context */
     memset(tcti_proxy, 0, sizeof(*tcti_proxy));
     TSS2_TCTI_MAGIC (tctiContext) = TCTI_PROXY_MAGIC;
-    TSS2_TCTI_VERSION (tctiContext) = TCTI_VERSION;
+    TSS2_TCTI_VERSION (tctiContext) = 2;
```

Comment:
Why?

Reply:
Because TCTI_VERSION is something internal to the tss2-tcti thing.
If anyone updates the tss2-tcti implementations to a version 3 I cannot expect them to know about this testcase that needs to implement the new function as well.
Thus, since I'm implementing v2 of the struct, I'm setting the version to 2...

Reply:
Shouldn't this be:
TSS2_TCTI_VERSION (tctiContext) = TCTI_PROXY_VERSION;

Reply:
Of course. You are right...
I'll fix this first thing tomorrow in the office...